From d5d960b0b8db73ffe4cff9d156b0883876d722ac Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 20 Jan 2023 18:29:13 +0100
Subject: [PATCH] bootctl: drop full stop at end of --help texts

We never do that, don't do so here either.

(cherry picked from commit 2b197967bf251ecf58b93fed0f51b9d4cd83fda4)

Related: RHEL-16354
---
 src/boot/bootctl.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/boot/bootctl.c b/src/boot/bootctl.c
index cc35147376..681c5bd44f 100644
--- a/src/boot/bootctl.c
+++ b/src/boot/bootctl.c
@@ -1455,8 +1455,8 @@ static int help(int argc, char *argv[], void *userdata) {
                "  is-installed        Test whether systemd-boot is installed in the ESP\n"
                "  random-seed         Initialize random seed in ESP and EFI variables\n"
                "\n%3$skernel Commands:%4$s\n"
-               "  kernel-identify     Identify kernel image type.\n"
-               "  kernel-inspect      Prints details about the kernel.\n"
+               "  kernel-identify     Identify kernel image type\n"
+               "  kernel-inspect      Prints details about the kernel\n"
                "\n%3$sOptions:%4$s\n"
                "  -h --help            Show this help\n"
                "     --version         Print version\n"
